debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresDefaultValueConverterIT.java

```diff
@@ -121,6 +122,46 @@ public void testShouldHandleDefaultValueFunctionsWithSchemaPrefixes() throws Exc
         assertThat(defaultValue).isEqualTo("00000000-0000-0000-0000-000000000000");
     }
 
+    @Test
+    @FixFor("DBZ-7562")
+    public void shouldTruncateDefaultValuePrecisionToMatchColumnMaxPrecision() throws Exception {
+        final String ddl = "DROP SCHEMA IF EXISTS s1 CASCADE;"
+                + "CREATE SCHEMA s1;"
+                + "CREATE TABLE s1.dbz7562 (pk SERIAL, cost numeric(12,0) not null default 0.0, PRIMARY KEY(pk));"
+                + "INSERT INTO s1.dbz7562 (cost) values (2.25);"
+                + "INSERT INTO s1.dbz7562 (cost) values (3);"
+                + "INSERT INTO s1.dbz7562 (cost) values (0);"
+                + "INSERT INTO s1.dbz7562 default values;";
+
+        // Test Snapshot
+        TestHelper.execute(ddl);
+
+        Configuration config = TestHelper.defaultConfig().build();
+        start(PostgresConnector.class, config);
+
+        waitForStreamingRunning("postgres", TestHelper.TEST_SERVER);
+
+        // Test Streaming
+        TestHelper.execute("INSERT INTO s1.dbz7562 (cost) values (5.5);");
+        TestHelper.execute("INSERT INTO s1.dbz7562 (cost) values (4);");
+        TestHelper.execute("INSERT INTO s1.dbz7562 (cost) values (0);");
+        TestHelper.execute("INSERT INTO s1.dbz7562 default values;");
+
+        final int expectedRecords = 8;
+        final SourceRecords records = consumeRecordsByTopic(expectedRecords);
+        final List<SourceRecord> recordsForTopic = records.recordsForTopic(topicName("s1.dbz7562"));
+        assertThat(recordsForTopic).hasSize(expectedRecords);
+
+        for (SourceRecord record : recordsForTopic) {
+            final Struct after = ((Struct) record.value()).getStruct(Envelope.FieldName.AFTER);
```

Comment:
For MySQL default valut tests we call `VerifyRecord.isValid` - it check serialization too. I recommend to add here. When done feel free to merge the PR yourself.